lib/ohai/runner.rb

```diff
@@ -45,8 +45,6 @@ def run_plugin(plugin)
           case plugin.version
           when :version7
             run_v7_plugin(plugin)
-          when :version6
-            run_v6_plugin(plugin)
```

Comment:
we'd be better off just raising unless this is a v7 plugin, and ditching this whole case statement i think.